google-cloud-storage/src/main/java/com/google/cloud/storage/Acl.java

```diff
@@ -294,13 +389,40 @@ public Role role() {
   }
 
   /**
-   * Returns an Acl object.
+   * Returns a builder for this {@code Acl} object.
+   */
+  public Builder toBuilder() {
+    return new Builder(this);
+  }
+
+  /**
+   * Returns an {@code Acl} object.
    *
    * @param entity the entity for this ACL object
    * @param role the role to associate to the {@code entity} object
    */
   public static Acl of(Entity entity, Role role) {
-    return new Acl(entity, role);
+    return builder(entity, role).build();
+  }
+
+  /**
+   * Returns a builder for {@code Acl} objects.
+   *
+   * @param entity the entity for this ACL object
+   * @param role the role to associate to the {@code entity} object
+   */
+  public static Builder builder(Entity entity, Role role) {
+    return new Builder(entity, role);
+  }
+
+  @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(this)
```

Comment:
Is there some reason that you use different orderings in different places?  L54-L57, L68-L71, & L421-L425